src/test/java/tests/HowItWorksTest.java

```diff
@@ -0,0 +1,26 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.HowItWorksPage;
+
+public class HowItWorksTest extends BaseTest {
+
+    HomePage homePage = new HomePage(driver);
+    HowItWorksPage page = new HowItWorksPage(driver);
+
+    @BeforeTest
+    public void howItWorksPage(){
+        homePage.openHomePage()
+                .goToHowItWorks();
```

Comment:
It's better use "BeforeTest" method for the general steps for all test, in your case you can use it for opening first page.